### DISCUSSION TITLE
English version

### DISCUSSION THREAD

**Discussion by tiendungdev** [Safe3/uusec-waf#74] [General]
English version please

**Comment by Safe3** [comment]
Should be available in the future

**Reply by tiendungdev** [reply]
Thank you bro!

**Comment by Safe3** [comment]
The English version is available now, you can try it!

**Comment by donnykalamuddin** [comment]
cannot login using username is "admin", and the default password is "Passw0rd!".

it, tables not being created on database from waf.sql.

**Reply by Safe3** [reply]
Host version or docker version?

**Reply by Safe3** [reply]
The problem had been fixed, you can try again!

**Comment by TacitTactics** [comment]
how do we accomplish this in the free community edition " enable dynamic passwords to ensure the security of WAF management backend accounts". I do not see the option the gui. Thank you

**Reply by Safe3** [reply]
The UUSEC WAF OTP funtion is what you need

**Comment by dnoizeshakerz** [comment]
default password, #Passw0rd does not work.
